azure-pipelines.yml

```diff
@@ -0,0 +1,15 @@
+jobs:
+  - template: common-pipelines.yml
+    parameters:
+      name: Linux
+      vmImage: ubuntu-1604
+
+  - template: common-pipelines.yml
+    parameters:
+      name: Windows
+      vmImage: vs2017-win2016
+
+  - template: common-pipelines.yml
+    parameters:
+      name: macOS
+      vmImage: xcode9-macos10.13
```

Comment:
is there any image with docker installed?

Reply:
I've checked before azure pipeline only support this one macos vm image which is not installed docker. See the link: https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-Readme.md